to-do-list/src/App.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useMemo, useEffect, useCallback } from 'react';
+import '../src/components/UI//styles/App.css'
+import { PostItemProps } from './components/Interfaces/interfaces';
+import PostList from './components/PostList/PostList';
+import PostForm from './components/PostForm/PostForm';
+import SearchBar from './components/SearchBar/SearchBar';
+import MyModal from './components/UI/modal/MyModal';
+import PostService from './API/PostService';
+import Weather from './components/Weather/Weather';
+
+
+function App() {
+  
+  const [posts, setPosts] = useState<PostItemProps[]>([])
+  const [filter, setFilter] = useState({query: ''})
+  const [modal, setModal] = useState(false)
```

Comment:
By the "modal" name I can think that you'll store tsx or modal object here, but not the bool value. For bool, use is/has prefix - isShowModal